ERCS/erc-7776.md

```diff
@@ -0,0 +1,572 @@
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in 
+RFC 2119 and RFC 8174.
+
+All Transparent Financial Statements Standard implementations MUST implement ERC-20 to represent shares, and the financial numbers such as Revenue, Costs of Goods Sold, Operating Expenses, Operating 
+Income, EBITDA, Other Income and Expenses, Net Income and Earnings Per Share MUST be displayed in the value of the protocol's stablecoin of choice.
+
+All Transparent Financial Statements MUST implement ERC-20's optional metadata extensions.
+The `name` and `symbol` functions SHOULD reflect the underlying token's `name` and `symbol` in some way.
+
+All methods MUST be of visibility `external`.
+
+All methods MUST return their financial numbers valued in the provided `stablecoin`.
+
+If the contract owner uses data or methods from other owned smart contracts external to their smart contract implementation of this standard, those smart contracts MUST be verified in the correspondent 
+blockchain explorer and of open and visible source code.
+
+_Timestamp Constraint_: For all methods, `startTimestamp` MUST be less than or equal to `endTimestamp`. If `startTimestamp` is equal to `endTimestamp`, the method returns a balance sheet snapshot. If 
+`startTimestamp` is less than `endTimestamp`, the method returns an income statement for that period.
+
+_Output Structs_: Instead of a single `uint256` value, each method returns a `struct` with one or _OPTIONAL_ more `uint256` entries to allow for detailed financial data, each one with their own customized 
+entry `name`.
+
+### Definitions:
+
+- Currency: The individual stablecoin used to value the publicly displayed financial numbers.
+- Revenue: Total earnings from selling products or services before expenses.
+- Cost of Goods Sold (COGS): Direct costs for producing goods/services, including labor and materials.
+- Operating Expenses: Expenses like Selling, General, and Administrative, Research and Development, and other operational costs.
+- Operating Income: Revenue minus operating expenses.
+- EBITDA: Earnings Before Interest, Taxes, Depreciation, and Amortization.
+- Other Income and Expenses: Non-operating income, such as interest, investment gains or losses.
+- Net Income: Profit after all expenses, taxes, and deductions.
+- EPS: Earnings per Share Token (ERC-20), showing profit allocated per share.
+
+### Methods
+
+#### `stablecoinAddress`
+
+Returns the `address` of the individual stablecoin used to value the publicly displayed financial numbers.
+
+```yaml
+- name: stablecoinAddress
+  type: function
+  visibility: external
+  stateMutability: view
+  inputs:
+    /none/
```

Comment:
```suggestion
```

I think you can just omit inputs if there are none, but I am far from an expert on this YAML format.